packages/patternfly-4/react-core/src/components/Button/Button.md

```diff
@@ -62,3 +62,23 @@ LinkButton = () => (
   </React.Fragment>
 );
 ```
+
```

Comment:
Can  you please add the example to the  "Link Button" section  a Button with `variant="link"` and ` component=“a”` and and 'href prop.  See the example in the issue.